src/main/java/io/phasetwo/service/resource/OrganizationsResource.java

```diff
@@ -142,4 +161,114 @@ public Response createOrg(@Valid Organization body) {
             session.getContext().getUri().getAbsolutePathBuilder().path(o.getId()).build())
         .build();
   }
+
+  @GET
+  @Path("export")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response exportOrgs(
+      @QueryParam("exportMembersAndInvitations") Boolean exportMembersAndInvitations) {
+    log.debugf("Export org for %s", realm.getName());
+    if (!auth.hasManageOrgs()) {
+      throw new NotAuthorizedException("Insufficient permission to export organization.");
+    }
+
+    var organizations =
+        orgs.searchForOrganizationStream(realm, Map.of(), 0, Integer.MAX_VALUE, Optional.empty())
+            .map(
+                organization ->
+                    KeycloakOrgsExportConverter
+                        .convertOrganizationModelToOrganizationRepresentation(
+                            organization, exportMembersAndInvitations))
+            .toList();
+
+    KeycloakOrgsRepresentation keycloakOrgsRepresentation = new KeycloakOrgsRepresentation();
+    keycloakOrgsRepresentation.setOrganizations(organizations);
+
+    Response.ResponseBuilder response = Response.ok();
+    response.type(MediaType.APPLICATION_JSON);
+    response.entity(keycloakOrgsRepresentation);
+
+    return response.build();
+  }
+
+  @POST
+  @Path("import")
+  @Produces(MediaType.APPLICATION_JSON)
+  @Consumes(MediaType.APPLICATION_JSON)
+  public Response importOrgs(
+      KeycloakOrgsRepresentation keycloakOrgsRealmRepresentation,
+      @QueryParam("skipMissingMember") Boolean skipMissingMember,
+      @QueryParam("skipMissingIdp") Boolean skipMissingIdp) {
```

Comment:
What are the defaults?

Reply:
The import is set by default to strict:
skipMissingMember=false,
skipMissingIdp=false

Thanks for the headup.